modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/search/index/BaseWorkflowMetricsIndexer.java

```diff
@@ -171,6 +175,55 @@ protected void setModuleServiceLifecycle(
 		ModuleServiceLifecycle moduleServiceLifecycle) {
 	}
 
+	protected void updateDocuments(
+		long companyId, Document fieldsDocument, Query filterQuery) {
```

Comment:
Hey @MarinhoFeliphe, why did you create a new method instead of changing the existing one? Thx